tutor/src/screens/grading-templates/index.js

```diff
@@ -33,6 +34,14 @@ class GradingTemplatesScreen extends React.Component {
     this.store.fetch();
   }
 
+  componentDidMount() {
+    document.body.classList.add('light');
+  }
+
+  componentWillUnmount() {
+    document.body.classList.remove('light');
+  }
```

Comment:
@nathanstitt Curious to hear what you think about this approach. At one point I looked into using Hooks instead, but had issues integrating it. Feels like there could be a better way though!